tools/model_partition_tool/README.md

```diff
@@ -0,0 +1,112 @@
+# Heuristic Graph Partitioning
+
+This folder contains the necessary scripts to perform a a heuristic-based graph partitioning for machine learning models.
+
+The main contents of this folder are as follows:
+
+- [Python Files](#python-scripts)
+- [How to Run Partitioning Algorithm?](#how-to-partition-tflite-model)
+
+
+## Python Scripts
+The python scripts in this folder require **Python3** to retrieve TFLite model operations. Additionally, please ensure that the python `numpy` package has been installed beforehand. The scripts also import the following modules: `queue`, `json` and `argparse`, all of which should be available by default. If not, please install them either by `pip install <package>` or `sudo apt install python-<package>`.
```

Comment:
@dr-venkman Do we need tensorflow package?

Reply:
The script uses the tfLite Interpreter to fetch operation and tensor details to produce a adjacency matrix. The partitioning algorithm relies on the weighted adjacency matrix to produce the splits.

Reply:
The script runs locally on development environment, so the requirement for TF package shouldn't be too restrictive. Please let me know your view.

Reply:
@dr-venkman I do not think it's too rectitive. I mean "please add tensorflow(+verison) info into readme.md for users" :)

Reply:
@chunseoklee , got it :), I have revised readme.md accordingly. Thanks for the comment.